.devcontainer/post_start.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# This script assumes we are running in a Miniconda container where:
+# - /opt/conda is the Miniconda installation directory
+# - https://github.com/conda/conda is mounted at /workspaces/conda
+# - https://github.com/conda/conda-libmamba-solver is mounted at
+#   /workspaces/conda-libmamba-solver
```

Comment:
You also use `mamba` itself, if available. Add to the comments here?

Reply:
Good point! Done!